test-docker.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
 set -ex
 PROJECT_ROOT="$(pwd)"
+TEST_ROOT="${PROJECT_ROOT}/tests/integration"
 TIMEOUT_SECONDS=600
 RUN_ID="$(date +%y%m%d_%H%M%S)"
+LOG_DIR="${PROJECT_ROOT}/logs/${RUN_ID}"
+while [ -d "$LOG_DIR"]; do
+    tmp_id="$((${tmp_id}+1))"
+    new_run="${RUN_ID}_${tmp_id}"
+    LOG_DIR="${PROJECT_ROOT}/logs/${RUN_ID}"
+done
+RUN_ID=''
+
+mkdir -p "${PROJECT_ROOT}/logs"
+mkdir "$LOG_DIR"
 
-BUILD_LOG="logs/${RUN_ID}_build.txt"
-RUN_LOG="logs/${RUN_ID}_run.txt"
-TEST_LOG="logs/${RUN_ID}_test.txt"
+BUILD_LOG="${LOG_DIR}/build.log"
```

Comment:
Is there a requirement for attaching a time stamp for the `build.log`, `run_master.log`, and `test.log`? For example, `build-2023-08-31-08-55-06.log`

Reply:
Now it is not attaching time stamp to each log file name. Instead, it creates a sub-directory with the name as timestamp and create all logs for that run inside the sub-directory.

Reply:
Ok